test/jte-runtime-test-kotlin/pom.xml

```diff
@@ -69,6 +68,22 @@
                 </plugins>
             </build>
         </profile>
+        <!-- Kotlin versions -->
```

Comment:
I don't like having this profile in two places, but I thought it was better than moving it to the parent pom.